src/test/rgw/test_multi.py

```diff
@@ -398,11 +398,11 @@ def init(parse_args):
             if is_master_zg and is_master:
                 if bootstrap:
                     # create admin user
-                    arg = ['--display-name', '"Zone User"', '--system']
+                    arg = ['--display-name', '"Zone User"', '--system', '--caps', '"roles=*"']
                     arg += admin_creds.credential_args()
                     admin_user.create(zone, arg)
                     # create test user
-                    arg = ['--display-name', '"Test User"']
+                    arg = ['--display-name', '"Test User"', '--caps', '"roles=*"']
```

Comment:
looks good for local testing with `test_multi.py` 👍 

for teuthology, we don't use this `test_multi.py` to drive the tests, and instead rely on `qa/tasks/rgw_multisite.py` and `qa/tasks/rgw_multisite_tests.py`

this test user is created [in rgw_multisite_tests.py](https://github.com/ceph/ceph/blob/master/qa/tasks/rgw_multisite_tests.py#L74-L75), so we'll need the same `--caps` argument there too

if the admin user also needs these caps, that user is created [in rgw_multisite.py](https://github.com/ceph/ceph/blob/master/qa/tasks/rgw_multisite.py#L129-L134)

Reply:
@cbodley : Ok, are the changes only adding caps to the correct user? Or do I need to add a separate test case also for teuthology?

Reply:
your changes to `rgw_multisite_tests.py` should be sufficient - that runs the same test cases (under `rgw_multi/`) that `test_multi.py` script, the only difference is in deployment